MODULE.bazel

```diff
@@ -67,6 +67,7 @@ single_version_override(
     patch_strip = 1,
     patches = [
         "@com_google_protobuf//:Disable_bundle_install.patch",
```

Comment:
Nit: can we move these to a /patches/rules_ruby directory?

Reply:
Will do in a follow up.